dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -8,6 +8,9 @@ RUN apt-get install -f -y ifupdown bridge-utils libdbus-1-3 libdaemon0 libjansso
 ## TODO: implicitly install dependencies
 RUN apt-get -y install libjemalloc1
 
+# Dependencies for sonic-cfggen                                                                                                                                                           
+RUN apt-get install -y python-lxml python-jinja2 python-netaddr python-ipaddr python-yaml 
```

Comment:
several mega each. We have discussed about putting those into docker-base, and the decision is to keep it as it is by now.

---
In reply to: [99411007](https://github.com/Azure/sonic-buildimage/pull/238#discussion_r99411007) [](ancestors = 99411007)